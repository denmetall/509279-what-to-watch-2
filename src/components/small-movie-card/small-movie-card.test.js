import React from 'react';
import renderer from 'react-test-renderer';
import SmallMovieCard from "./small-movie-card.jsx";

it(`Components SmallMovieCard renders correctly`, () => {
  const tree = renderer
    .create(<SmallMovieCard movieTitle={`some title`} onClickTitleHandler={jest.fn()}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
