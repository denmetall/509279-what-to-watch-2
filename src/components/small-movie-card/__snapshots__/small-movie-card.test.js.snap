// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components SmallMovieCard renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      muted={true}
      poster="img/johnny-english.jpg"
      width="280"
    >
      <source
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      />
    </video>
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/1"
      onClick={[Function]}
    >
      Drive
    </a>
  </h3>
</article>
`;
